Userland/Libraries/LibWebView/OutOfProcessWebView.cpp

```diff
@@ -298,11 +298,11 @@ void OutOfProcessWebView::notify_server_did_middle_click_link(Badge<WebContentCl
         on_link_middle_click(url, target, modifiers);
 }
 
-void OutOfProcessWebView::notify_server_did_start_loading(Badge<WebContentClient>, const AK::URL& url. bool is_redirect)
```

Comment:
This `.` -> `,` change should be done in the relevant commit

Also since the parameter is unused until then you might not want to give the parameter a name until this commit

Reply:
@davidot  After some rebasing trouble I think I have applied all your comments:^)